### DISCUSSION TITLE
Welcome to Jimmi Discussions!

### DISCUSSION THREAD

**Discussion by p-fruck** [Music-Bot-for-Jitsi/Jimmi#5] [Announcements]
## 👋 Welcome!
  Jimmi is using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vanillacode314** [comment]
how to play music? the bot joins but how to make it play something?

**Reply by p-fruck** [reply]
Have you installed the browser extension? The old installation process (which is required since we are currently working on a proper re-implementation of the project) is documented [here](https://github.com/Music-Bot-for-Jitsi/Jimmi/tree/v1.0.0#rocket-get-started). As soon as the extension is installed and the bot joins the meeting, you can use the chat commands `!play <url | searchTerm>` and `!queue <url | searchTerm>` to play songs and add them to the queue.
